packages/psss-server/src/routes/country/FetchCountrySites.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { getSortByKey } from '@tupaia/utils';
+import { Route } from '../Route';
+
+export class FetchCountrySites extends Route {
+  async buildResponse() {
+    const { fields } = this.req.query;
+    const { countryCode } = this.req.params;
+
+    const sites = await this.entityConnection.fetchSites(countryCode, { fields });
+    const data = sites.sort(getSortByKey('name'));
```

Comment:
This wouldn't be needed after https://github.com/beyondessential/tupaia-backlog/issues/2658 - could you add a TODO comment?